### PR TITLE
Improved `all` feature

### PR DESCRIPTION
**Pull request by jho5245**
Improved `all` feature when buying and selling items

### CODE REVIEW

File: src/main/java/org/maxgamer/quickshop/Shop/ShopManager.java [hunk 1]
```diff
@@ -441,16 +447,27 @@ private void actionTrade(@NotNull Player p, @NotNull HashMap<UUID, Info> actions
...
+                        amount = Util.countSpace(p.getInventory(), shop.getItem());
+                    }
+                    // typed 'all', check if player has enough money than price * amount
+                    double price = shop.getPrice();
+                    double balance = eco.getBalance(p.getUniqueId());
+                    amount = Math.min(amount, (int) Math.floor(balance / price));
+                    if (amount < 1) {
+                        // when typed 'all' but player can't buy any items
+                        // @TODO: from getConfig().getString("shop.not-enough-money-for-trade-all-items");
+                        p.sendMessage("you don't have enough money!");
```
Comment (Ghost-chu) [inline src/main/java/org/maxgamer/quickshop/Shop/ShopManager.java]: All messages must use language file, please use MsgUtils.getMessage()

Comment (jho5245) [reply src/main/java/org/maxgamer/quickshop/Shop/ShopManager.java]: changed to use MsgUtil.getMessage()

**Comment by jho5245** [comment]
> not-enough-money-for-trade-all-items not vaild phrase in the messages file.
> Edit it to `you-cant-afford-to-buy`.
> MsgUtil.getMessage("you-cant-afford-to-buy", totalprice, playerHadMoney)
> And you files not passed the Travis-CI check

**Comment by Ghost-chu** [comment]
Travis-CI build check failed, some errors you might need to fix.

**Comment by Ghost-chu** [comment]
When you ready for merge, comment "ok" under this pr and i will merge it

**Comment by jho5245** [comment]
ok, successfully work done. also implemented minimum-price feature.

**Comment by Ghost-chu** [comment]
Thanks for your hard work, new binary file you should can found in my BinaryRepo in 5min

**Pull request merged**